web/frontend/src/components/slam/index.svelte

```diff
@@ -69,6 +69,7 @@ let motionPath: Float32Array | undefined;
 let mappingSessionStarted = false;
 let isLocalizingMode: boolean | undefined;
 let lastReconfigured: Timestamp | undefined;
+let mappingMode = 'undefined';
```

Comment:
```suggestion
let mode: ValueOf<typeof slamApi.MappingMode> = slamApi.MappingMode.MAPPING_MODE_UNSPECIFIED
```

`ValueOf` will need to be imported from `typefest`, which will need to be installed since we don't yet have it in RDK:
```ts
import type { ValueOf } from 'type-fest';
```
```bash
npm i -D type-fest
```